earth_enterprise/src/fusion/autoingest/AssetFactory_unittest.cpp

```diff
@@ -366,6 +412,227 @@ TEST_F(AssetFactoryTest, FindMakeAndUpdateAssets)
     ASSERT_EQ(handle_4->needed, true);
 }
 
+class MockVersionDType : public MockMutableAssetVersion
```

Comment:
This is more a matter of personal preference, but I would put this class at the top with the other mock class definitions.